### PR TITLE
Adding github actions for CI

### PR DESCRIPTION
**Pull request by n3wscott**
Signed-off-by: Scott Nichols <snichols@vmware.com>

### CODE REVIEW

**Comment by slinkydeveloper** [comment]
CircleCI is not cool anymore uhm? :smile4

**Comment by slinkydeveloper** [comment]
> {"level":"error","ts":1600301604.6655478,"logger":"fallback","caller":"client/client.go:204","msg":"Error while opening the inbound connection: dial tcp: lookup 9300aec18cb1: Temporary failure in name resolution","stacktrace":"github.com/cloudevents/sdk-go/v2/client.(*ceClient).StartReceiver.func2\n\t/home/runner/work/sdk-go/sdk-go/v2/client/client.go:204"}

Weird one, sounds like a networking problem

**Comment by n3wscott** [comment]
> CircleCI is not cool anymore uhm? 😄

ClircleCI is making it very hard for contributors to view the failure logs. Github actions is faster (like 2-3x faster) than CircleCI because it is run in parallel.

Still sorting out the integration tests...

**Review by slinkydeveloper — APPROVED** [review]

File: .github/workflows/integration.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      matrix:
+        go-version: [1.13.x, 1.14.x]
+        platform: [ubuntu-latest]
+
+    runs-on: ${{ matrix.platform }}
+
+    services:
+      kafka:
+        image: lensesio/fast-data-dev
+        ports:
```
Comment (slinkydeveloper) [inline .github/workflows/integration.yaml:23]: That's why it doesn't work

**Review by slinkydeveloper — APPROVED** [review]

**Pull request merged**